ekino/newrelic-bundle/2.0/config/packages/ekino_new_relic.yaml

```diff
@@ -0,0 +1,6 @@
+ekino_new_relic:
+    enabled: false
+    application_name: 'My Symfony Application' # Remove this to use the value from php.ini
+    api_key: "%env(NEWRELIC_API_KEY)%"
```

Comment:
I suggest to add a comment like `# This configuration is only required when using the "newrelic:notify-deployment"`

Reply:
Yeah, but what if the bundle adds a new feature that uses the API key? 
I do not want to update the recipe then, right?

Or am I overthinking it?